src/core/server/http/lifecycle/on_pre_response.ts

```diff
@@ -68,11 +81,13 @@ const preResponseResult = {
  * @public
  */
 export interface OnPreResponseToolkit {
+  render: (html: string) => OnPreResponseResult;
```

Comment:
1. Let's name it consistently - body.
2. we support different types of `body` https://github.com/elastic/kibana/blob/feceb0f98eb817f065834f8b6c9c628cee41383a/src/core/server/http/router/response.ts#L81 can be done in a separate PR

Reply:
Fixed no 1.